paimon-core/src/main/java/org/apache/paimon/Snapshot.java

```diff
@@ -357,6 +362,16 @@ public static Snapshot fromJson(String json) {
     public static Snapshot fromPath(FileIO fileIO, Path path) {
         try {
             return Snapshot.fromJson(fileIO.readFileUtf8(path));
```

Comment:
Here my throw UncheckedIOException, could you add same logic to handle the UncheckedIOException.

Reply:
Where to throw `UncheckedIOException`?

Reply:
In Snapshot#fromPath will call Snapshot#fromJson, and then call JsonSerdeUtil.fromJson.
In JsonSerdeUtil.fromJson, IOException will transform to UncheckedIOException which is not IOException.

Reply:
We don't need to take care it, it is true exception, just throw it.